core/iwasm/libraries/libc-wasi/sandboxed-system-primitives/src/posix.c

```diff
@@ -778,6 +779,7 @@ fd_object_get_locked(struct fd_object **fo, struct fd_table *ft, __wasi_fd_t fd,
                      __wasi_rights_t rights_inheriting)
     TRYLOCKS_EXCLUSIVE(0, (*fo)->refcount) REQUIRES_EXCLUSIVE(ft->lock)
 {
+    printf("[SSP] fd_object_get_locked\n");
```

Comment:
These printf will be removed in the future, right?
And had better use os_printf or LOG_XXX instead if needed, printf may be not supported in some platforms.